GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_with_metacal_griz.yaml

```diff
@@ -0,0 +1,8 @@
+subclass_name: composite.CompositeReader
+description: DC2 Run 2.1i Object Catalog with metacal (griz)
+catalogs:
+  - catalog_name: dc2_object_run2.1i_dr1b_all_columns
+    matching_method: 'id'
+  - catalog_name: dc2_metacal_griz_run2.1i_dr1b
+    matching_method: 'id'
```

Comment:
change to `MATCHING_FORMAT`

Reply:
I'm keeping the matching by ID as it's not significantly slower, much safer, and handles missing patches from metacal